source/install/desktop.rst

```diff
@@ -192,18 +193,19 @@ For additional troubleshooting tips, see the `troubleshooting guide <https://www
 Reporting Issues
```

Comment:
This section should be worked on sometime in the future because it's still referencing the old developer toosl and might be confusing to people tryin to report issues.

Reply:
Thanks, @svelle. I see what you mean. I don't have any experience with this troubleshooting process but if you have the content on hand and have the bandwidth to add it, that'd be awesome.